### PR TITLE
Remove unused features

### PR DESCRIPTION
**Pull request by julianbraha**
[[PR Description]]

This removes unused features from various crates. This was likely caused by refactors that removed the code that was using these features, without removing the features.

### CODE REVIEW

File: crates/theme_importer/Cargo.toml [hunk 1]
```diff
@@ -23,5 +23,5 @@ serde.workspace = true
 serde_json.workspace = true
 simplelog = "0.9"
 strum = { version = "0.25.0", features = ["derive"] }
-theme = { path = "../theme", features = ["importing-themes"] }
```
Comment (maxdeviant) [inline crates/theme_importer/Cargo.toml:26]: The use of the `importing-themes` feature here is intended, so we should keep it.

It's used to signify that the `theme_importer` is importing themes.

Comment (julianbraha) [reply crates/theme_importer/Cargo.toml]: Got it, I've added it back

File: crates/audio/Cargo.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@ collections = { path = "../collections" }
 util = { path = "../util" }
 
 
-rodio ={version = "0.17.1", default-features=false, features = ["wav"]}
```
Comment (osiewicz) [inline crates/audio/Cargo.toml:19]: Hmm, I believe we need that feature as the sounds from `assets` are all in .wav format.

Comment (osiewicz) [reply crates/audio/Cargo.toml:19]: Yeah, I've just checked and this would break notification sounds in Zed.

Comment (julianbraha) [reply crates/audio/Cargo.toml:19]: Thanks for confirming. I think the rodio crate should change how they use features, but regardless, I should have done my due diligence on this one. I've added it back.

**Comment by mikayla-maki** [comment]
Given the changes that would have broken core zed features and the lack of a compelling developer UX or user facing justification, I'm going to say we close this PR for now.

**Pull request closed without merging**